openassessment/staffgrader/serializers/assessments.py

```diff
@@ -19,10 +19,11 @@ class SubmissionDetailFileSerilaizer(serializers.Serializer):
 class AssessmentPartSerializer(serializers.ModelSerializer):
     name = serializers.CharField(source='criterion.name')
     option = serializers.CharField(source='option.name', default=None)
+    points = serializers.IntegerField(source='option.points', default=None)
```

Comment:
Is it possible to _not_ have a `points` field? Is the default wanted/necessary?

Reply:
Yes, if we have a feedback-only criterion, this will be None